.github/workflows/tests.yml

```diff
@@ -68,11 +75,25 @@ jobs:
           # Wait for OSDs to become up
           sleep 30
 
+      - name: Enable RGW
+        run: |
+          set -eux
+          sudo microceph enable rgw
+          # Wait for RGW to settle
+          sleep 30
```

Comment:
I'm not a big fan of these sleeps to assert that a service is up. Would it be too hard to loop until the RGW has settled (up or down) ?

Reply:
I just tried to stay consistent with the existing tests :-D 
But yea I kinda feel the same way, I'll try to do some retry loop